client/python/lib/test/test_tensors.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+from tensorflow.core.framework.tensor_shape_pb2 import *
+from tensorflow.core.framework.tensor_pb2 import TensorProto
+from ovmsclient.tfs_compat.grpc.tensors import make_ndarray
+import numpy as np
+
+shape = TensorShapeProto(dim = [TensorShapeProto.Dim(size=3)])
+
+def test_make_ndarray_valid_int():
+    tensor_proto = TensorProto(tensor_shape=shape, dtype = 6, tensor_content = bytes([1,2,3]))
+    array = make_ndarray(tensor_proto)
+    assert array.tolist() == [1, 2, 3]
+    assert array.dtype == np.int8
+
+def test_make_ndarray_valid_float():
+    hex_string = "0000803f0000004000004040"
```

Comment:
Is it possible we start with array and then eventually go through hex? 
It's hard to quickly tell that this value means [1, 2, 3] in float32.

Same applies to below cases.